doc/howto/cluster_form.md

```diff
@@ -377,19 +377,21 @@ opyQ1VRpAg2sV2C4W8irbNqeUsTeZZxhLqp4vNOXXBBrSqUCdPu1JXADV0kavg1l
 
 ## Use MicroCloud
 
-```{youtube} https://www.youtube.com/watch?v=ZSZoLnp-Ip0
+```{youtube} https://www.youtube.com/watch?v=iWZYUU8lX5A
 ```
 
-Instead of setting up your LXD cluster manually, you can use [MicroCloud](https://snapcraft.io/microcloud) to get a fully highly available LXD cluster with Ceph storage up and running.
+Instead of setting up your LXD cluster manually, you can use [MicroCloud](https://microcloud.is/) to get a fully highly available LXD cluster with OVN and with Ceph storage up and running.
```

Comment:
@ru-fu looks like the steps on https://microcloud.is/ are also missing microovn too :)

Reply:
Yep, Miona is aware :)